core/src/test/java/io/grpc/internal/AbstractTransportTest.java

```diff
@@ -1219,9 +1210,6 @@ public void clientCancel() throws Exception {
     assertNotNull(clientStreamListener.trailers.get(TIMEOUT_MS, TimeUnit.MILLISECONDS));
     Status serverStatus = serverStreamListener.status.get(TIMEOUT_MS, TimeUnit.MILLISECONDS);
     assertNotEquals(Status.Code.OK, serverStatus.getCode());
-    // Cause should not be transmitted between client and server
```

Comment:
Deleting these lines concerns me. Is this necessary?

Reply:
You're right, not necessary! I will add the assertion back in